code/obj/item/stickers.dm

```diff
@@ -107,6 +108,12 @@
 			attached.visible_message("<span class='alert'><b>[src]</b> is destroyed!</span>")
 		..()
 
+	mouse_drop(atom/over_location)
```

Comment:
should probably rename to `over_object`, to match parent argument names (which also has an `over_location`, but as the *third* argument)

Reply:
Oh, oop